fork.html

```diff
@@ -0,0 +1,170 @@
+            <li><strong>✅ Wait on HashiCorp's response</strong>. We reached out to HashiCorp publicly and privately and requested a response by August 25th.</li>  
+            <li><strong>✅ Start working on the OpenTF fork</strong>. With no response from HashiCorp, we created the OpenTF fork, and started working on it in private.</li>  
+            <li><strong>✅ Apply to join the Linux Foundation</strong>. We want OpenTF to be part of an impartial, community-driven foundation, so we submitted all the paperwork to join the Linux Foundation.</li>  
+            <li><strong>✅ Open up community discussions</strong>. We opened up <a href="https://github.com/orgs/opentffoundation/discussions">OpenTF Discussions on GitHub</a> to give the community a way to have discussions, provide feedback, ask questions, etc.</li>  
+            <li><strong>✅ Release the OpenTF fork preview</strong>. As per this announcement, we are making the OpenTF fork preview public at <a href="https://github.com/opentffoundation/opentf">github.com/opentffoundation/opentf</a>!</li>
+          </ul>
+          <h4 class="small-margin">What's currently in progress</h4>          
+          <ul class="tight-list">
+            <li><strong>🔄 Finalize project name</strong>. We need to decide whether to stick with "OpenTF" as a name or pick a different name.</li>  
+            <li><strong>🔄 Finalize foundation & processes</strong>. We need to finalize the details of the OpenTF foundation, including the processes we will use for the steering committee, RFCs, accepting contributions, etc.</li>  
+            <li><strong>🔄 Create initial OpenTF Registry</strong>. HashiCorp recently made some (unannounced) <a href="https://github.com/opentffoundation/roadmap/issues/24#issuecomment-1699535216">changes</a> to the terms of the Terraform Registry, saying it may only be used with Terraform. As a stopgap solution to unblock our develpoment and testing, we are launching an initial OpenTF Registry. We'll develop the official OpenTF Registry solution via an official RFC process later.</li>  
+            <li><strong>🔄 Release process</strong>. Put in place a process for creating OpenTF releases.</li>  
+            <li><strong>🔄 Alpha release</strong>. Once the above items are done, we will create the first OpenTF release. This will be an alpha release, meant for testing by the community.</li>  
+          </ul>
+          <h4 class="small-margin">What's coming soon</h4>          
+          <ul class="tight-list">
+            <li><strong>⏳Start accepting community PRs and RFCs</strong>. Once all the details of the foundation, registry, and release process are in place, we will start accepting contributions from the community!</li>  
+            <li><strong>⏳Create an official OpenTF Registry via an RFC process</strong>. Go through an RFC process to create the official OpenTF Registry solution (replacing the stopgap solution).</li>  
```

Comment:
I wouldn't call it so heavily a "stopgap" solution. Please just call it the "current temporary solution".

That is because through the RFC process we might decide that the "current temporary solution" is actually the way to go long-term. And we wouldn't like to call the way to go long-term a "stopgap solution"...

Reply:
Updated to call it the "initial" solution.